### PR TITLE
allow consecutive startPranks

### LINKED ISSUE #389: Display the proof execution time
When running `kontrol prove`, after the execution ends, it displays each proof's status. It would also be helpful to print the execution time of each proof. The same thing applies to the `kontrol list` command; it would be helpful to print the execution time of each proof.

### PR DESCRIPTION
**Pull request by anvacaru**
Closes #389

This updates the `#setPrank` rule to fix a bug for which the execution gets stuck when a new `startPrank` is called while a prank is already active.

### CODE REVIEW

File: src/tests/integration/test-data/foundry/running-tests.log
Comment (PetarMax) [inline src/tests/integration/test-data/foundry/running-tests.log:1]: Empty file?

File: src/kontrol/kdist/cheatcodes.md [hunk 1]
```diff
@@ -1328,7 +1328,7 @@ Will also return true if REASON is `.Bytes`.
            <prevOrigin> _ => OG </prevOrigin>
            <newCaller> _ => NEWCALLER </newCaller>
            <newOrigin> _ => NEWORIGIN </newOrigin>
-           <active> false => true </active>
+           <active> _ => true </active>
```
Comment (PetarMax) [inline src/kontrol/kdist/cheatcodes.md:1331]: Yes, so this will allow the next prank to override the previous one rather than not fire.

Comment (anvacaru) [reply src/kontrol/kdist/cheatcodes.md:1331]: Yes. Currently it gets stuck.

File: src/tests/integration/test-data/foundry/test/PlainPrankTest.t.sol [hunk 2]
```diff
@@ -97,6 +97,15 @@ contract PlainPrankTest is Test {
         vm.stopPrank();
     }
 
+    function test_startPrank_consecutive() public {
+        AdditionalToken token = new AdditionalToken();
+        vm.startPrank(address(0));
+        token.incrementCount();
+        vm.startPrank(address(1));
+        vm.stopPrank();
+        assert(token.count() == 1);
```
Comment (PetarMax) [inline src/tests/integration/test-data/foundry/test/PlainPrankTest.t.sol:106]: Could we have an assertion about the pranking address here, just to see it's working properly?

**Review by PetarMax — APPROVED** [review]

**Pull request merged**